x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -267,6 +267,20 @@ export class Simulator {
     this.sideEffectSimulator.controls.provideAnimationFrame();
   }
 
+  /**
+   * The last value written to the clipboard via the `SideEffectors`.
+   */
+  public get clipboardText(): string {
+    return this.sideEffectSimulator.controls.clipboardText;
+  }
+
+  /**
+   * Call this to resolve the promise returned by the `SideEffectors` `writeText` method (which in production points to `navigator.clipboard.writeText`.
+   */
+  confirmTextWrittenToClipboard(): void {
```

Comment:
Writing to the clipboard is asynchronous. Calling this will resolve the next enqueue promise returned from `writeTextToClipboard`